mnefun/_mnefun.py

```diff
@@ -1792,19 +1792,20 @@ def _viz_raw_ssp_events(p, subj, ridx):
                 if ri in p.get_projs_from]
     all_proj = op.join(pca_dir, 'preproc_all-proj.fif')
     projs = read_proj(all_proj)
-    ev_names = [op.join(pca_dir + '/' + ii) for ii in
-                ['preproc_ecg-eve.fif', 'preproc_blink-eve.fif']]
+    ev_names = glob.glob(op.join(pca_dir, 'preproc*-eve.fif'))
```

Comment:
we can probably leave the old check, it's a bit more explicit than globbing. Users for example might reasonable rename old versions in some way that the `glob` would catch.

Then  later, instead of doing the `assert len(ev) == 2`, you can do two separate `if op.isifle(...)` checks.

Does that sound alright?